backend/src/types.ts

```diff
@@ -245,67 +244,152 @@ export type BuildStatus = {
   timestamp?: string;
 };
 
+export type EnvironmentVariable = {
```

Comment:
types and rest calls should be added to the api documentation from this PR. https://github.com/opendatahub-io/odh-dashboard/pull/246. once it goes in.  I would create a new issue for that and do it in another PR.

Reply:
Will do 👍